Project/include/config.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+
+define("DB_HOSTNAME",'localhost');
```

Comment:
you probably don't want to recreate this config file. Just stick with the one you've already got. There are two reasons: 1) even if you have multiple projects in one file system, you're always connecting to the same database (by the way, I'd suggest using different branches in git for different projects rather than making them all their own directories unless you really want them all hosted on your website at the same time)

2) This file needs to have the real database password and stuff like that on the real server. In other words, it contains sensitive info, and that sensitive info is different on your local server vs. your remote one. For these reasons, you don't want to commit this file into git. The normal config file is already set up to not go into git, but this new one you made isn't.